cmd/libs/go2idl/args/args.go

```diff
@@ -37,10 +37,12 @@ import (
 // Default returns a defaulted GeneratorArgs. You may change the defaults
 // before calling AddFlags.
 func Default() *GeneratorArgs {
-	return &GeneratorArgs{
+	generatorArgs := &GeneratorArgs{
 		OutputBase:       DefaultSourceTree(),
 		GoHeaderFilePath: filepath.Join(DefaultSourceTree(), "k8s.io/kubernetes/hack/boilerplate/boilerplate.go.txt"),
 	}
+	generatorArgs.AddFlags(pflag.CommandLine)
```

Comment:
What if a user doesn't use the `Default()` to generate args? So IMO calling the `AddFlags` in `Execute` is better.